find_unknown_value.rb

```diff
@@ -1,13 +1,17 @@
 require "damerau-levenshtein"
 
-def find_unknown_command(known_commands, unknown_command)
+# TODO We use this function for packages and ax commands, is this the best guess distance "best" for both cases?
```

Comment:
I can reuse this for both command discovery and package discovery, so make generic